### PR TITLE
Add bourbon support

### PR DESCRIPTION
**Pull request by rosswarren**
http://bourbon.io/

Not yet fixed the tests, but putting up the PR now so you can see what I am up to in case I am taking the wrong approach.

### CODE REVIEW

File: tasks/wrappers/sass.js [hunk 1]
```diff
@@ -38,7 +39,13 @@ Sass.prototype.file = function(fileObj, outputStyle){
                 log.warn('Sass Error');
                 reject(e);
             }
-        });
+        };
+
+        if (options.bourbon === true) {
+            renderOptions.includePaths = bourbon.includePaths;
```
Comment (peter-mouland) [inline tasks/wrappers/sass.js]: by adding a sass config object to `caddy.config.js` you can add includePaths:
```javascript
tasks:{ 
  build:['sass'],
},
sass: {
  includePaths: 'path/to/stuff'
}
```

File: tasks/wrappers/sass.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ var path = require('path');
 var fs = require('../utils/fs');
 var File = require('../utils/file');
 var log = require('../utils/log');
+var bourbon = require('node-bourbon');
```
Comment (peter-mouland) [inline tasks/wrappers/sass.js]: i dont know that bourbon is needed as we use auto-prefixer to sort out any browser prefixes.
Would you be able to include bourbon sass files within the scss files directly with an inludePath setup?

Comment (rosswarren) [inline tasks/wrappers/sass.js]: Yeah I don't think it is actually needed now I see how little it is doing. I will close :smile:

File: tasks/wrappers/sass.js (hunk 1 above)
Comment (rosswarren) [inline tasks/wrappers/sass.js]: Thanks for the tip!

**Comment by peter-mouland** [comment]
@rosswarren just release v2.0.0-beta.4 which allows full Sass options set via the caddy.config.js file.
I dont know what other Sass setting there are, but now you can set as many as you want!

**Comment by peter-mouland** [comment]
for completeness here is the auto prefixer page: https://github.com/postcss/autoprefixer
and a list of Sass options : https://github.com/sass/node-sass
The latest version (to be released tonight) of the caddy will be updated to have all up to date plugins

**Pull request closed without merging**